core/src/main/java/io/camunda/migrator/converter/DecisionInstanceConverter.java

```diff
@@ -0,0 +1,79 @@
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.migrator.converter;
+
+import static io.camunda.migrator.impl.util.ConverterUtil.convertDate;
+import static io.camunda.migrator.impl.util.ConverterUtil.getNextKey;
+
+import io.camunda.db.rdbms.write.domain.DecisionInstanceDbModel;
+import io.camunda.migrator.impl.util.ConverterUtil;
+import io.camunda.search.entities.DecisionInstanceEntity;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.camunda.bpm.engine.history.HistoricDecisionInputInstance;
+import org.camunda.bpm.engine.history.HistoricDecisionInstance;
+import org.camunda.bpm.engine.history.HistoricDecisionOutputInstance;
+
+public class DecisionInstanceConverter {
+
+  public DecisionInstanceDbModel apply(HistoricDecisionInstance decisionInstance,
+                                       Long decisionDefinitionKey,
+                                       Long processDefinitionKey,
+                                       Long decisionRequirementsDefinitionKey,
+                                       Long processInstanceKey,
+                                       Long rootDecisionDefinitionKey,
+                                       Long flowNodeInstanceKey,
+                                       String flowNodeId) {
+    Long decisionInstanceKey = getNextKey();
+    return new DecisionInstanceDbModel.Builder()
+        .partitionId(ConverterUtil.C7_HISTORY_PARTITION_ID)
+        .decisionInstanceId(String.format("%d-%s", decisionInstanceKey, decisionInstance.getId()))
+        .decisionInstanceKey(decisionInstanceKey)
+        .state(DecisionInstanceEntity.DecisionInstanceState.UNSPECIFIED) // TODO not stored in C7
+        .evaluationDate(convertDate(decisionInstance.getEvaluationTime()))
+        .evaluationFailure(null) // not stored in HistoricDecisionInstance
+        .evaluationFailureMessage(null) // not stored in HistoricDecisionInstance
+        .result(String.valueOf(decisionInstance.getCollectResultValue()))
+        .flowNodeInstanceKey(flowNodeInstanceKey)
+        .flowNodeId(flowNodeId)
+        .processInstanceKey(processInstanceKey)
+        .processDefinitionKey(processDefinitionKey)
+        .processDefinitionId(decisionInstance.getProcessDefinitionKey())
+        .decisionDefinitionKey(decisionDefinitionKey)
+        .decisionDefinitionId(decisionInstance.getDecisionDefinitionKey())
+        .decisionRequirementsKey(decisionRequirementsDefinitionKey)
+        .decisionRequirementsId(decisionInstance.getDecisionRequirementsDefinitionKey())
+        .rootDecisionDefinitionKey(rootDecisionDefinitionKey)
+        .decisionType(DecisionInstanceEntity.DecisionDefinitionType.UNSPECIFIED) // TODO would need to check xml?
+        .tenantId(StringUtils.isEmpty(decisionInstance.getTenantId()) ? "<default>" : decisionInstance.getTenantId())
```

Comment:
📓 Depending on our discussion, let's add a test case for decision deployed with tenant?